src/modules/postcode-lookup/index.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Returns a postocde lookup module.
+ * @param {Object} window  Global window object
+ */
+function createPostcodeLookup(window) {
+    // Define an empty JSON object to temporarily store matched address results.
+    // eslint-disable-next-line no-unused-vars
+    let tmpAddressSearchResultsJson = {};
+
+    function addSearchResultsToSelectElement(data) {
+        const selectElementResults = window.document.getElementById(
```

Comment:
cache this outside of the `addSearchResultsToSelectElement` function. Saves querying the DOM each time the function is called.